config/database.yml.example.production

```diff
@@ -1,19 +1,19 @@
 staging:
   adapter: mysql2
   encoding: utf8
-  database: <%= ENV['RES_DB_NAME'] %>
```

Comment:
These are pulling in the values themselves, not just checking them as booleans, so they shouldn't be replaced.